pkg/epinio/components/application/AppSource.vue

```diff
@@ -443,10 +465,14 @@ export default Vue.extend<Data, any, any, any>({
         <h3>{{ t('epinio.applications.steps.source.gitUrl.url.label') }}</h3>
         <LabeledInput
           v-model="gitUrl.url"
+          v-focus
```

Comment:
At some point this needs to be added to the first input field of the other types as well